### ISSUE TITLE
Separate addon for auto play/pause media

### ISSUE THREAD

**Issue by Noitidart** [carlin-q-scott/browser-media-keys#87]
The "Automatically play/pause media" feature is really awesome. Just food for thought, to make it more visible/discover-able, what about creating a separate addon featuring this?

**Comment by carlin-q-scott** [comment]
It's an interesting idea and not terribly difficult to do but I'm not a huge fan add-on-add-ons. Some people hate this feature and some people love it so it does make sense to make it its own thing.

**Comment by Noitidart** [comment]
Yeah it's a tricky thing. We don't want to make super tiny addons with like super minute features. But at same time we don't want to make super fat addons with so many features it hides the others (TabMix Plus like).

I personally didn't realize it was Media Keys that was doing this. I had youtube videos in random tabs. And I only realized it was media keys as I looked at the options page earlier today haha.

I searched "how to disable youtube videos from auto playing in other tabs" and got no results a month or so ago, was baffled until now haha.
